.github/workflows/tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Node tests
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test-main:
+    name: Run lib tests
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup node
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14.x
+
+    - run: yarn install --frozen-lockfile
+    - run: yarn test
+
+  test-example-in-local:
```

Comment:
Hi @pdpino , I'm wondering if we need this on the workflow?

Reply:
@hoangnm you mean not testing the app by itself. Sounds good as well

Reply:
yeah, in my view, the flow should run the tests of the library only.